.github/workflows/citation.yml

```diff
@@ -0,0 +1,72 @@
+name: Citation preview
```

Comment:
We technically don't need this check right now, since we don't have a `CITATION.bib` file, yet.
But we have kept this check around in the other addons, too.

Essentially this will provide a quick preview of the `CITATION.bib` file being used.